crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -486,81 +491,1014 @@ mod YASPoolTests {
+                    tick_lower: min_tick,
+                    tick_upper: max_tick,
+                    amount: 1,
+                    data: ArrayTrait::<felt252>::new()
+                );
+        }
+
 
         mod FailureCases {
-            // TODO: 'fails if tickLower greater than tickUpper'
+            use super::{
+                setup, MIN_TICK, MAX_TICK, tick_spacing, FeeAmount, fee_amount,
+                IERC20DispatcherTrait, FACTORY_ADDRESS, TOKEN_A, TOKEN_B, WALLET,
+                encode_price_sqrt_1_1
+            };
+            use yas_core::numbers::signed_integer::{
+                i32::i32, i32::i32_div_no_round, integer_trait::IntegerTrait
+            };
+            use yas_core::contracts::yas_pool::{
+                YASPool, YASPool::ContractState, YASPool::YASPoolImpl, YASPool::InternalImpl,
+                IYASPool, IYASPoolDispatcher, IYASPoolDispatcherTrait
+            };
+            use yas_core::contracts::yas_router::{
+                YASRouter, IYASRouterDispatcher, IYASRouterDispatcherTrait
+            };
+
+
+            #[test]
+            #[available_gas(2000000000)]
+            #[should_panic(expected: ('TLU', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_fails_tick_lower_greater_than_tick_upper() {
+                let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                mint_callback
+                    .mint(
+                        yas_pool.contract_address,
+                        WALLET(),
+                        IntegerTrait::<i32>::new(1, false),
+                        IntegerTrait::<i32>::new(0, false),
+                        1
+                    );
+            }
+
+
+            #[test]
+            #[available_gas(2000000000)]
+            #[should_panic(expected: ('TLM', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_fails_tick_lower_than_min() {
+                let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                mint_callback
+                    .mint(
+                        yas_pool.contract_address,
+                        WALLET(),
+                        IntegerTrait::<i32>::new(887273, true),
+                        IntegerTrait::<i32>::new(0, false),
+                        1
+                    );
+            }
+
+
+            #[test]
+            #[available_gas(2000000000)]
+            #[should_panic(expected: ('TUM', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_fails_tick_greater_than_max() {
+                let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                mint_callback
+                    .mint(
+                        yas_pool.contract_address,
+                        WALLET(),
+                        IntegerTrait::<i32>::new(0, false),
+                        IntegerTrait::<i32>::new(887273, false),
```

Comment:
What does this value represent?

Reply:
it's MAX_TICK + 1 , it is hardcoded on univ3 tests, but now I changed it to more accurately represent the value